app/controllers/reports_controller.rb

```diff
@@ -1,10 +1,10 @@
 class ReportsController < ApplicationController
-  before_action :logged_in_user, only: %i(create new index destroy)
+  before_action :logged_in_user, only: %i(create new index destroy show)
```

Comment:
```suggestion
  before_action :logged_in_user, except: %i(edit update)
```
Nhiều hơn 4 thì except cho ngắn ạ